### PR TITLE
ROX-13231: use passed context when non-postgres

### PR DESCRIPTION
**Pull request by keyallis**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: central/graphql/resolvers/cluster_vulnerabilities.go [hunk 1]
```diff
@@ -312,10 +312,9 @@ func withOpenShiftTypeFiltering(q string) string {
 		search.NewQueryBuilder().AddExactMatches(search.CVEType, storage.CVE_OPENSHIFT_CVE.String()).Query())
 }
 
-func (resolver *clusterCVEResolver) clusterVulnerabilityScopeContext() context.Context {
+func (resolver *clusterCVEResolver) clusterVulnerabilityScopeContext(ctx context.Context) context.Context {
 	if resolver.ctx == nil {
-		log.Errorf("attempted to scope context on nil")
```
Comment (md2119) [inline central/graphql/resolvers/cluster_vulnerabilities.go:317]: 😱

File: central/graphql/resolvers/nodes.go [hunk 2]
```diff
@@ -543,24 +518,23 @@ func (resolver *nodeResolver) PlottedNodeVulnerabilities(ctx context.Context, ar
...
+	res.ctx = resolver.nodeScopeContext(ctx)
 	return res, nil
 }
 
 func (resolver *nodeResolver) UnusedVarSink(_ context.Context, _ RawQuery) *int32 {
 	return nil
 }
 
-func (resolver *nodeResolver) nodeScopeContext() context.Context {
+func (resolver *nodeResolver) nodeScopeContext(ctx context.Context) context.Context {
```
Comment (md2119) [inline central/graphql/resolvers/nodes.go:532]: Context of incoming requests is not expected to be nil. Therefore, for this and other scope context builders, check `ctx` is not nil and panic on dev build. That way we can catch if incorrect context is passed around. Remember that  `resolver.ctx` is set to `ctx` from request.

```
if ctx == nil {
  utils.Should("argument 'ctx' is nil")
}
```

**Review by md2119 — CHANGES_REQUESTED** [review]
One comment on nil check on `ctx` from request, otherwise lgtm.

File: central/graphql/resolvers/cluster_vulnerabilities.go (hunk 1 above)
Comment (keyallis) [reply central/graphql/resolvers/cluster_vulnerabilities.go:317]: Do we want to retain these messages? They don't seem necessary or at the very least shouldn't be errors

**Review by md2119 — APPROVED** [review]

**Pull request merged**